target/linux/ramips/image/mt7621.mk

```diff
@@ -1653,6 +1653,17 @@ define Device/sercomm_na502s
 endef
 TARGET_DEVICES += sercomm_na502s
 
+define Device/snr_snr-cpe-me2-lite
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  IMAGE_SIZE := 16064k
+  DEVICE_VENDOR := SNR
+  DEVICE_MODEL := SNR-CPE-ME2-Lite
+  UIMAGE_NAME := SNR-CPE-ME2-Lite
+  DEVICE_PACKAGES := kmod-mt7603 kmod-mt7615e kmod-mt7663-firmware-ap -kmod-mt7615-firmware
```

Comment:
`kmod-mt7615e` do not include `kmod-mt7615-firmware`, so `-kmod-mt7615-firmware` does nothing here.